lte/gateway/c/core/oai/common/common_types.h

```diff
@@ -177,9 +177,10 @@ typedef enum {
   RAT_HRPD           = 2001,
   RAT_UMB            = 2002,
   RAT_EHRPD          = 2003,
+  RAT_NG_RAN         = 1006,
```

Comment:
place this after RAT_EUTRAN to group 3GPP ones together.

Reply:
Done